disperser/dataapi/subgraph_client.go

```diff
@@ -51,6 +51,19 @@ type (
 		BlockNumber     uint64
 		TransactionHash []byte
 	}
+	OperatorQuorum struct {
+		Operator      string
+		QuorumNumbers []byte
+		BlockNumber   uint64
+	}
+	OperatorQuorumEvents struct {
+		// Mapping from operator address to a list of sorted events (ascending by BlockNumber)
+		// where the operator was added to quorums.
+		AddedToQuorum map[string][]*OperatorQuorum
```

Comment:
nit: start comment with the name of the variable. `AddedToQuorum is a mapping from...`

same below

Reply:
Done